### PR TITLE
Add GitHub Actions badge

### PR DESCRIPTION
**Pull request by rudeboybert**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -223,6 +225,11 @@ get_regression_points(score_model)
 
 ``` r
 library(broom)
```
Comment (ismayc) [inline README.md:227]: Might need to update packages?

Comment (rudeboybert) [reply README.md:227]: sorry, where? all my local packages are up to date

Comment (rudeboybert) [reply README.md:227]: sorry, missed this

Comment (rudeboybert) [reply README.md:227]: will update R

**Comment by rudeboybert** [comment]
@ismayc My bad on that previous pkg warning. I've reinstalled R, updated pkgs, ran styler, rebuilt the README, and rebuilt the pkgdown site.

**Pull request merged**